config/board.ini

```diff
@@ -270,3 +270,43 @@ lib_ignore =
 extra_scripts =
     ${config:smallNoI2s.extra_scripts}
     pre:./scripts/get_git_rev.py
+
+
+; ********************************************************************************
+; M5Stack Core ESP32 WiFi and Bluetooth Module Development Board
+; ********************************************************************************
+[board:m5stack_core]
+extends = mcu:esp32, display:m5stack_core, config:normal
+board = m5stack-grey
+board_build.partitions = default_8MB.csv
+board_build.filesystem = littlefs
+build_flags =
+    ${mcu:esp32.build_flags}
+    ${display:m5stack_core.build_flags}
+    -DBOARD_M5STACK 
+    -D BOARD_HAS_PSRAM

```

Comment:
I can't see that it has PSRAM!? But I might be wrong.

Reply:
The board does not have PSram (the red one https://shop.m5stack.com/products/fire-iot-development-kit?variant=16804798169178 does)